mmv1/products/databasemigrationservice/ConnectionProfile.yaml

```diff
@@ -266,25 +289,37 @@ properties:
     properties:
       - !ruby/object:Api::Type::String
         name: 'host'
-        required: true
+        required_with:
+        - 'postgresql.0.port'
```

Comment:
There's some lint errors on this line, could you fix them?

Reply:
Done.